packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -15,6 +15,17 @@ import { DOCROOT, wordPressSiteUrl } from './config';
 import { isUploadedFilePath } from './is-uploaded-file-path';
 import patchWordPress from './wordpress-patch';
 
+type PlaygroundStartupOptions = {
+  wpVersion?: string;
+  phpVersion?: string;
+};
+const startupOptions = parseWorkerStartupOptions<PlaygroundStartupOptions>();
+
+// Expect underscore, not a dot. Vite doesn't deal well with the dot in the
+// parameters names passed to the worker via a query string.
+const wpVersion = (startupOptions.wpVersion || '6_1').replace('_', '.');
+const phpVersion = (startupOptions.phpVersion || '8_0').replace('_', '.');
+
```

Comment:
This was moved up so that we can change the docroot depending on the WP version further down in this file (if that's even the right location).

For wp-develop the docroot needs to be `/wordpress/build`

Reply:
Here's what this could look like once the docroot is exported by the WordPress es module:

```ts
server.docroot = wpLoaderModule.docroot;
```

Docroot is readonly at the moment, but I'm happy to open it for writing as well.